examples/slo/main.tf

```diff
@@ -42,3 +42,28 @@ resource "chronosphere_slo" "slo" {
     }
   }
 }
+
+resource "chronosphere_slo" "slo_with_signal_grouping" {
+  name                   = "SLO With Signal Grouping"
+  collection_id          = chronosphere_collection.c.id
+  notification_policy_id = chronosphere_notification_policy.np.id
+
+  signal_grouping {
+    label_names       = ["label1", "label2"]
+    signal_per_series = true
```

Comment:
i don't think this is supposed to be compatible with label names -- it should be one or the other

maybe do 2 separate SLOs, one with `signal_per_series` and one with `label_names`, and let's double check that setting both results in an error (in the API, and in terraform via dry-run)

Reply:
it failed in internal scenario tests when i set them both so we are good there.